### ISSUE TITLE
In-line response schemas are not expanded

### ISSUE THREAD

**Issue by MordantWastrel** [coldbox-modules/cbSwagger#21]
The docs indicate that you can specify a file pointer in an in-line response annotation, e.g.

```
/**
  * @response-200 { "description" : "User successfully updated", "schema" : "/resources/apidocs/schema.json##user" }`
*/

This works, but only to populate the description field. The schema is not an expandable node in the resulting swagger docs when viewed in ReDoc. I tried a few other permutations of nested JSON and using $ref - no luck. 

Is this working as intended or should you be able to use a file pointer?

**Comment by jclausen** [comment]
@MordantWastrel - shouldn't your annotation be

```
@response-200 { "description" : "User successfully updated", "schema" : { "$ref" : "/resources/apidocs/schema.json##user" } }
```
?

**Comment by MordantWastrel** [comment]
Not according to the README: 

`@response-200 { "description" : "User successfully updated", "schema" : "/resources/apidocs/schema.json##user" }`

**Comment by jclausen** [comment]
Then that's a typo in the README.  If the value of the annotation is JSON, then it will be parsed and expanded as such. If it's a straight-up file path, then it just reads it in and parses.   Could you try the above and see if it fixes the issue?

**Comment by MordantWastrel** [comment]
This works, as you suggested:

`* @response-500 { "description" : "General Application Error", "schema" : { "$ref" : "/resources/api/standardError.json" } }`

This does not:

`     * @response-500 { "description" : "General Application Error", "schema" : { "$ref" : "/resources/api/standardError.json" } }

```
The $ref file pointer of ~standardError.json could not be loaded and parsed as a valid object. If your $ref file content is an array, please nest the array within an object as a named key.
```

Looks like it's just not using the new substitution.

**Comment by jclausen** [comment]
I have added support for usage of `~` samples path shortcuts to `$ref` objects inside annotations in the next release.  This will be available as of v3.0.0
